library/pk_internal.h

```diff
@@ -21,6 +21,20 @@
 #include "psa/crypto.h"
 #endif
 
+/* Headers/footers for PEM files */
```

Comment:
Ok, I understand the motivation behind the previous commit now, but we still can't change observable behaviour like that. We could define a new function with the new behaviour instead. There are probably some code size savings from sharing strings between pkparse and pkwrite, and those might be larger than the cost of the new function - this would have to be measured of course.